### PR TITLE
Remove ops from Layer Creator/ Node Converter - part 3

### PR DESCRIPTION
**Pull request by blesniewski**
Removing ops from layer creator/ node converter:

- Convert
- DepthToSpace
- MVN
- NormalizeL2
- NotEqual
- Subtract

42362

### CODE REVIEW

**Comment by blesniewski** [comment]
This is only a draft PR:
I want to run all the tests
Also waiting until PR 3226 is merged so I can use a function I'm introducing there.

File: ngraph/core/src/op/mvn.cpp [hunk 1]
```diff
@@ -49,6 +49,8 @@ op::MVN::MVN(const Output<Node>& data, AxisSet reduction_axes, bool normalize_va
     , m_reduction_axes{reduction_axes}
 {
     constructor_validate_and_infer_types();
+    const size_t chanelAxis = 1;
```
Comment (blesniewski) [inline ngraph/core/src/op/mvn.cpp:52]: This change I'm not 100% sure of, but it's the only path that we have a problem on:

1. _MVN_ is created by the first creator
2. It's then copied using _clone_with_new_inputs_
3. When running a second creator from _clone_with_new_inputs_ we lose this _m_across_channels_ flag. 

That is why I'm re-checking if it should be enabled.

Comment (ilyachur) [reply ngraph/core/src/op/mvn.cpp:52]: Do we have a test cases which show this issue?

**Review by ilyachur — COMMENTED** [review]
Please resolve merge conflicts

File: ngraph/core/src/op/mvn.cpp (hunk 1 above)
Comment (blesniewski) [reply ngraph/core/src/op/mvn.cpp:52]: Yeah, for example ReadMVNNetwork from IE functional tests was failing.

File: inference-engine/src/readers/ir_reader/ie_ir_parser.cpp [hunk 2]
```diff
@@ -480,9 +473,14 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     if (!ngraphNode && opsets.count(params.version)) {
         auto opset = opsets.at(params.version);
         std::string type = params.type;
+
         if (type == "Const") {
             type = "Constant";
         }
+        // MVN was missing in opset1
+        if (type == "MVN" && params.version == "opset1") {
```
Comment (ilyachur) [inline inference-engine/src/readers/ir_reader/ie_ir_parser.cpp:481]: Can we use one`if` for `MVN` and `ROIPooling`?

Comment (blesniewski) [reply inference-engine/src/readers/ir_reader/ie_ir_parser.cpp:481]: Sure!

File: ngraph/core/include/ngraph/op/not_equal.hpp [hunk 3]
```diff
@@ -49,6 +49,7 @@ namespace ngraph
 
                 bool evaluate(const HostTensorVector& outputs,
                               const HostTensorVector& inputs) const override;
+                virtual bool visit_attributes(AttributeVisitor& visitor) override;
```
Comment (ilyachur) [inline ngraph/core/include/ngraph/op/not_equal.hpp:52]: nit: virtual is redundant

**Review by ilyachur — APPROVED** [review]
I will merge this PR. But I propose to resolve a nitpick in the next PR

File: ngraph/core/include/ngraph/op/not_equal.hpp (hunk 3 above)
Comment (blesniewski) [reply ngraph/core/include/ngraph/op/not_equal.hpp:52]: I'll fix this in part 4 👍

**Pull request merged**